stable/airflow/templates/deployments-web.yaml

```diff
@@ -180,15 +180,15 @@ spec:
               echo 'installing requirements...' &&
               mkdir -p /usr/local/airflow/.local/bin &&
               export PATH=/usr/local/airflow/.local/bin:$PATH &&
-              /usr/local/scripts/install-requirements.sh &&
+              /usr/local/scripts/install-requirements.sh && {{ .Values.airflow.startupScript | trim | indent 14 }}
               echo 'executing webserver...' &&
               airflow webserver
```

Comment:
This should in my opinion be done after running the scheduler and not the webserver. Scripts are already ran after the initdb in the scheduler and it seems like the right place to me, especially if the scripts can interact with the DB (using the airflow command for example).